src/backend/app/jaxa/jaxa/settings.py

```diff
@@ -0,0 +1,92 @@
+# Scrapy settings for jaxa project
+#
```

Comment:
I think, you set up a Scrapy app(Jaxa), which is great! However, I highly recommend using Scrapy's **Core API**. It simplifies the workflow and is perfect for single-page scripts without requiring a full project setup.  

Here’s an example:  
```python
from scrapy.crawler import CrawlerRunner
from scrapy import Spider
from twisted.internet import reactor

class ExampleSpider(Spider):
    name = "example"
    start_urls = ["https://example.com"]

    def parse(self, response):
        print("Scraped content:", response.body[:100])

runner = CrawlerRunner()
runner.crawl(ExampleSpider)
runner.join().addBoth(lambda _: reactor.stop())
reactor.run()
```